sycl/test-e2e/Matrix/joint_matrix_bf16_fill_k_cache_impl.hpp

```diff
@@ -93,12 +93,13 @@ double joint_matmul(TOperand *A, TOperand *B, TResult *C, queue &q, int i) {
       // KCache2/prefRow = 32/8 = 4 SGs on the row dimension
 #ifdef VNNI
           // In the VNNI case, each subgroup still gets prefRow x prefCol
-          // subgroups distribution become 16 on the row dimension and
-          // 2 on the  col dimension
-          // PVC case: NCache2*2/prefCol = 16, KCache2/prefRow   so NumSGs*2
-          // warps are needed for each row (512)
+          // In the PVC case: subgroups distribution become
+          // NCache2*2/prefCol = 16 subgroups on the col dimension and
+          // (KCache2/2)/prefRow = 2 on the row dimension
+          // (NCache2*2)/prefCol = 512/32 = 16 SGs on the column dimension and
+          // (KCache2/2)/prefRow = 16/8 = 2 SGs on the row dimension
```

Comment:
nit: duplicated comments
```suggestion
          // (NCache2*2)/prefCol = 512/32 = 16 SGs on the column dimension and
          // (KCache2/2)/prefRow = 16/8 = 2 SGs on the row dimension
```